README.rst

```diff
@@ -16,6 +16,8 @@ This code runs on Python 2.7.
 1.  Get a local copy of this repo.
 
 2.  (Optional)  Create and activate a virtualenv to work in.
+        
+        A virtualenv allows you to isolate your development dependencies for this project from any other dependencies that you might have for other projects. If you do any tinkering at all, it will save you a lot of headaches to install a virtualenv to work in. Read more about it here: http://docs.python-guide.org/en/latest/dev/virtualenvs/
```

Comment:
Also, I'm not sure of the right indentation to use for this paragraph to be included in bullet 2.  I check these things by pasting the .rst text into http://rst.ninjs.org/